resources/js/Composables/useInfiniteScrolling.js

```diff
@@ -1,39 +1,39 @@
 import axios from 'axios'
-import {onMounted, onUnmounted, ref} from 'vue'
+import { onMounted, onUnmounted, ref } from 'vue'
 
 
 const isFetching = ref(false)
 const loadMoreIntersect = ref(null)
 export const useInfiniteScrolling = (slips) => {
-  const fetchSlips = async () => {
-    const next_url = slips.value.next_page_url
-
-    if (!next_url) return
-
-    isFetching.value = true
-    await axios.get(next_url)
-      .then((response) => {
-        slips.value.data.push(...response.data.data)
-        slips.value.next_page_url = response.data.next_page_url
-        console.log(slips.value.meta)
-        isFetching.value = false
-      })
-  }
-
-  const observer = new IntersectionObserver(entries => {
-    entries.forEach(entry => {
-      if (entry.isIntersecting) {
-        fetchSlips()
-      }
+    const fetchSlips = async () => {
+        const next_url = slips.value.next_page_url
```

Comment:
Wondering if this check can't be done above "fetchSlips", seems like unneeded code to run if next_page_url either way will stop it further ahead in callbacks